### ISSUE TITLE
Error opening Zip file from Update Mura Core  6.2.6460

### ISSUE THREAD

**Issue by wolffy** [blueriver/MuraCMS#2376]
This error occurs on all websites at the same hosting company. Not sure if it is a Mura or hosting issue.

string  error in opening zip file

Array
1  
Struct
codePrintHTML  
string  518: <br> 519: /\* Open Zip file _/<br> <b>520: zipFile.init(arguments.zipFilePath);</b><br> 521: <br> 522: /_ Zip file entries _/<br>
codePrintPlain  
string  518: 519: /_ Open Zip file _/ 520: zipFile.init(arguments.zipFilePath); 521: 522: /_ Zip file entries */
column  
number  0
id  
string  ??
line  
number  520
Raw_Trace  
string  zip_cfc$cf.udfCall(/home/protrade/public_html/requirements/mura/Zip.cfc:520)
template  
string  /home/protrade/public_html/requirements/mura/Zip.cfc
type  
string  cfml
2  
Struct
codePrintHTML  
string  112: &lt;!--- make sure that there are actually any updates---&gt;<br> 113: &lt;cfif len(diff)&gt;<br> <b>114: &lt;cfset rs=zipUtil.list(&quot;#currentDir##zipFileName#.zip&quot;)&gt;</b><br> 115: <br> 116: &lt;cfif directoryExists(&quot;#currentDir##zipFileName#&quot;)&gt;<br>
codePrintPlain  
string  112:  113: <cfif len(diff)> 114: <cfset rs=zipUtil.list("#currentDir##zipFileName#.zip")> 115: 116: <cfif directoryExists("#currentDir##zipFileName#")>
column  
number  0
id  
string  ??
line  
number  114
Raw_Trace  
string  autoupdater.autoupdater_cfc$cf.udfCall(/home/protrade/public_html/requirements/mura/autoUpdater/autoUpdater.cfc:114)
template  
string  /home/protrade/public_html/requirements/mura/autoUpdater/autoUpdater.cfc
type  
string  cfml
3  
Struct
codePrintHTML  
string  330: &lt;cftry&gt;<br> 331: &lt;cfif rc.$.validateCSRFTokens(context='updatecore')&gt;<br> <b>332: &lt;cfset updated=application.autoUpdater.update()&gt;</b><br> 333: &lt;cfset files=updated.files&gt;<br> 334: &lt;cfelse&gt;<br>
codePrintPlain  
string  330: <cftry> 331: <cfif rc.$.validateCSRFTokens(context='updatecore')> 332: <cfset updated=application.autoUpdater.update()> 333: <cfset files=updated.files> 334: <cfelse>
column  
number  0
id  
string  ??
line  
number  332
Raw_Trace  
string  admin.core.views.csettings.list_cfm$cf.call(/home/protrade/public_html/admin/core/views/csettings/list.cfm:332)
template  
string  /home/protrade/public_html/admin/core/views/csettings/list.cfm
type  
string  cfml
4  
Struct
codePrintHTML  
string  1535: &lt;/cfif&gt;<br> 1536: <br> <b>1537: &lt;cfsavecontent variable='response'&gt;&lt;cfinclude template=&quot;#viewPath#&quot;/&gt;&lt;/cfsavecontent&gt;</b><br> 1538: <br> 1539: &lt;cfreturn response /&gt;<br>
codePrintPlain  
string  1535: </cfif> 1536: 1537: <cfsavecontent variable='response'><cfinclude template="#viewPath#"/></cfsavecontent> 1538: 1539: <cfreturn response />
column  
number  0
id  
string  ??
line  
number  1537
Raw_Trace  
string  admin.framework_cfc$cf.udfCall7(/home/protrade/public_html/admin/framework.cfc:1537)
template  
string  /home/protrade/public_html/admin/framework.cfc
type  
string  cfml
5  
Struct
codePrintHTML  
string  546: <br> 547: if ( structKeyExists(request, 'view') ) {<br> <b>548: out = internalView( request.view );</b><br> 549: } else {<br> 550: out = onMissingView( request.context );<br>
codePrintPlain  
string  546: 547: if ( structKeyExists(request, 'view') ) { 548: out = internalView( request.view ); 549: } else { 550: out = onMissingView( request.context );
column  
number  0
id  
string  ??
line  
number  548
Raw_Trace  
string  admin.framework_cfc$cf.udfCall2(/home/protrade/public_html/admin/framework.cfc:548)
template  
string  /home/protrade/public_html/admin/framework.cfc
type  
string  cfml

**Comment by mattlevine** [comment]
Unfortunately, the auto updater is turned off for Mura 6 right now. There was an issue resulting in updating people to Mura 7 unexpectedly.  You can still autoupdate to Mura 7 by setting autoupdatemode=preview in your settings.ini.cfm and reloading and auto updating once more.

However, It's actually fairly easy to manually update.

Updating Mura's core:
1. Back everything up.
2. Download the latest Mura standard from getmura.com or from the 6.2 branch on github.
3. Unzip the download and copy everything except the "default" directory and past it over your Mura root.
4. Copy the /config/settings.ini.cfm and config/cfapplication.cfm(if you added an customizations) from your backup into you {Mura root}/config. directory.
5. Make a request to your Mura root with your appreload key and applydbupdate key.

http://yourdomain.com/?appreload&applydbupdates

Update a Mura site.
1. Back the siteID directory up.
2. Download the latest Mura standard from getmura.com or from the 6.2 branch on github.
3. Unzip the downloaded zip to your desktop.  
4. Rename the the siteID directory to {siteid}_temp
5. Copy the "default" directory into your Mura root and rename it to the your siteID that is being updated.
6. Copy the follow files and directory from your {siteid{_temp directory into the new created {siteid} directory:

{siteid}_temp/includes/contentRenderer.cfc
{siteid}_temp/includes/eventHandler.cfc
{siteid}_temp/includes/Application.cfc
{siteid}_temp/includes/themes

Basically everything in the ..includes directory except the display_objects directory.

And then the custom directory inside the display_objects direcory:
{siteid}_temp/includes/display_objects/custom
        
7. Reload Mura.
